src/vaadin-menu-bar-sub-menu-mixin.html

```diff
@@ -138,6 +140,9 @@
     }
 
     _close() {
+      if (this._activeButton.hasAttribute('expanded')) {
```

Comment:
let's add check for `if (this._activeButton && this._activeButton.hasAttribute('expanded'))`

Otherwise the error is thrown when clicking button without sub-menu:

```console
vaadin-menu-bar-sub-menu-mixin.html:153 Uncaught TypeError: Cannot read property 'hasAttribute' of undefined
    at HTMLElement._close (vaadin-menu-bar-sub-menu-mixin.html:153)
```

And we should also check if submenu is opened before closing it (but not really related to this PR)

Reply:
Add checks for `_activeButton` and `_subMenu.opened`.